app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx

```diff
@@ -51,6 +51,62 @@ export class CloudProviderIcon extends React.Component<IServerGroupHeaderProps>
   }
 }
 
+export interface IImageListState {
+  collapsed: boolean;
+}
+
+export class ImageList extends React.Component<IServerGroupHeaderProps, IImageListState> {
+  private toggle() {
+    this.setState((previousState: IImageListState) => {
+      return { collapsed: !previousState.collapsed };
+    });
+  }
+
+  constructor(props: IServerGroupHeaderProps) {
+    super(props);
+    this.state = {
+      collapsed: true,
+    };
+    this.toggle = this.toggle.bind(this);
+  }
+
+  public render() {
+    const images = this.props.images.sort();
+    const { collapsed } = this.state;
+
+    return (
+      <>
+        {collapsed && (
+          <>
+            <span>{images[0]}</span>
+            &nbsp;
+            {images.length > 1 && (
+              <a href="javascript:void()" onClick={this.toggle}>
```

Comment:
I think it might be semantically more appropriate to use a button here since the anchor isn't referencing anything.  Deck provides for a link-style button with the following:

```ts
<button className="link" onClick={this.toggle}>
  (+ {images.length - 1} more)
</button>
```

This may also require a little further styling to get padding & font size squared away.

Reply:
@sbwsg makes sense! Were you thinking of any specifics in terms of styling ? 
(color, padding, .. ?)

Reply:
Only for it to match the neighbouring text as closely as possible. Same font-size, font, line-spacing etc as the image name text next to it.  It may be that the `link` classname does all of that in the context of this header.

Reply:
Good catch! I think the only thing that had to be adjusted was the fontSize. Let me know if you want to change the color to match as well. But I'm assuming we want to keep the colors associated with the `link` style.